### PR TITLE
Add svg feedback icons

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr replaces the buttons with text with svg icons.

https://screenshot.googleplex.com/RioGd686QA96utP.png

### CODE REVIEW

File: src/main/webapp/view/index.html [hunk 1]
```diff
@@ -33,10 +33,71 @@ <h2 id="header-text">ZoomTube</h2>
...
+              height="2.5em"
+              viewBox="0 0 16 16"
+              class="bi bi-skip-backward"
+              fill="currentColor"
+              xmlns="http://www.w3.org/2000/svg"
+              focusable="false"
+            >
+              <path
+                fill-rule="evenodd"
+                d="M.5 3.5A.5.5 0 0 1 1 4v3.248l6.267-3.636c.52-.302 1.233.043 1.233.696v2.94l6.267-3.636c.52-.302 1.233.043 1.233.696v7.384c0 .653-.713.998-1.233.696L8.5 8.752v2.94c0 .653-.713.998-1.233.696L1 8.752V12a.5.5 0 0 1-1 0V4a.5.5 0 0 1 .5-.5zm7 1.133L1.696 8 7.5 11.367V4.633zm7.5 0L9.196 8 15 11.367V4.633z"
```
Comment (billie-cat) [inline src/main/webapp/view/index.html:97]: Idk what these things mean. Can you create consts in HTML to store these values? If not, then I would add a comment.

**Review by billie-cat — CHANGES_REQUESTED** [review]

File: src/main/webapp/view/index.html (hunk 1 above)
Comment (f-camp) [reply src/main/webapp/view/index.html:97]: +1 Please add a comment describing what this is.

**Review by f-camp — CHANGES_REQUESTED** [review]

File: src/main/webapp/view/index.html (hunk 1 above)
Comment (jaysaleh) [reply src/main/webapp/view/index.html:104]: Added comment!

Comment (billie-cat) [reply src/main/webapp/view/index.html:104]: These are all different paths in your comment can you explain which icon it is? Example:

**Review by billie-cat — CHANGES_REQUESTED** [review]

**Review by billie-cat — APPROVED** [review]

File: src/main/webapp/view/index.html (hunk 1 above)
Comment (jaysaleh) [reply src/main/webapp/view/index.html:104]: Added

**Comment by jaysaleh** [comment]
Needed to merge, so made changes. Had to dismiss review to merge.

**Pull request merged**